apps/stress-test/src/components/v8/StressApp.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import { getTestParams } from '../../shared/testParams';
+import { performanceMeasure } from '../../shared/performanceMeasure';
+import { StressContainer } from './StressContainer';
+
+export const StressApp = () => {
+  const [numChildren, setNumChildren] = React.useState(getTestParams().numStartNodes);
```

Comment:
nit: Is there a reason you need to call getTestParams() multiple times? Could you just do something like this?
```ts
const { test, numStartNodes, numAddNodes, numRemoveNodes } = getTestParams();
```